core/src/main/scala/cats/data/ContT.scala

```diff
@@ -188,6 +190,58 @@ object ContT {
   def later[M[_], A, B](fn: => (B => M[A]) => M[A]): ContT[M, A, B] =
     DeferCont(() => FromFn(AndThen(fn)))
 
+  /*
+   * Limits the continuation of any inner [[shiftT]]
+   */
+  def resetT[M[_]: Monad: Defer, A, B](contT: ContT[M, A, A]): ContT[M, B, A] =
```

Comment:
can we reverse `B` and `A` here so that we return `ContT[M, A, B]` as in other cases?